sources/models/shared-defaults/defaults.toml

```diff
@@ -46,16 +46,24 @@ restart-commands = ["/bin/systemctl try-restart host-containerd.service"]
 version-lock = "latest"
 ignore-waves = false
 
+[services.thar-be-updates]
+configuration-files = ["thar-be-updates-toml"]
+restart-commands = []
+
 [services.updog]
 configuration-files = ["updog-toml"]
 restart-commands = []
 
+[configuration-files.thar-be-updates-toml]
+path = "/etc/thar-be-updates.toml"
+template-path = "/usr/share/templates/thar-be-updates-toml"
+
 [configuration-files.updog-toml]
 path = "/etc/updog.toml"
 template-path = "/usr/share/templates/updog-toml"
 
 [metadata.settings.updates]
-affected-services = ["updog"]
+affected-services = ["updog", "thar-be-updates"]
```

Comment:
If you scope the config file to just `settings.updates.version-lock` then you can attach the affected services metadata to that single setting.

It's OK to leave it like this though since `thar-be-updates` may need other update settings in the future and it's pretty annoying to shift this stuff around.